tensorflow/python/ops/parallel_for/BUILD

```diff
@@ -146,7 +146,7 @@ cuda_py_test(
         "//tensorflow/python:util",
     ],
     shard_count = 5,
-    tags = ["optonly"],  # Too slow in non-opt mode
+    tags = ["optonly", "no_rocm"],  # Too slow in non-opt mode
```

Comment:
Other than batch matmul broadcasting test, are other tests also failing? I'd worry about a wildcard like this would hide other issues while I'm working on a fix for batch matmul test.

Let's not disable the entire math_test, but only the `test_batch_matmul_broadcast` test. This is so that we still have good test coverage among other math operations.

Reply:
@jerryyin I've verified the other sub-tests were also failed with the same signature, that's why I disabled the test.

Reply:
Strange, that shouldn't happen. The "broadcasting not supported" error message should only happen in batch-matmultiply related tests. It includes at most two test cases: `test_batch_matmul_broadcast` and `test_batch_matmul`. If something else is also failing, we need to take a look at it.

What are the other tests that also fails? Could you double confirm they fail for the exact same reason?

Reply:
@jerryyin , quite some other subtests failed with the identical signature. We can re-enable this test in your coming PR that introduces the fixes if the weekly merge gets in first. Otherwise, I can cherry-pick your fix and re-enable the test case here.

Reply:
@sunway513 That works too. I'll remember to re-enable this one back.